recipes/bwa/0.7.3a/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: bwa
+  version: "0.7.3a"
+
+build:
+  number: 0
+
+source:
+  fn: bwa-0.7.3a.tar.bz2
+  url: http://downloads.sourceforge.net/project/bio-bwa/bwa-0.7.3a.tar.bz2
+  md5: 7ed9f5d750d26f5bf91cee1fa6f65df2
+
+requirements:
+  build:
+    - gcc
+    - zlib
```

Comment:
zlib should be pinned if possible
And it would be nice to use GCC only for Linux

Reply:
Sorry I didn't catch this earlier, my linter isn't as sharp.  Do I need to do an update PR or is this fine to leave as is?

Reply:
I think it is fine for the moment. But please keep this in mind. zlib already broke ABI once.

Reply:
what version would you pin to? How would you use gcc only for linux? we can update all of the bwa builds at once after that

Reply:
https://github.com/conda-forge/conda-forge.github.io/blob/0661e2ecf4b213ddf36350ca100a5a820eb17eb7/scripts/pin_the_slow_way.py#L85 :)

Reply:
Here is the GCC-linux only logic: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/samtools/meta.yaml#L18

That version of zlib might also be acceptable, not 100% sure what is best.

Reply:
nvm, I see how to only use gcc for linux.